airbyte-integrations/connectors/source-bing-ads/metadata.yaml

```diff
@@ -39,6 +39,11 @@ data:
           primary keys. A data reset and schema refresh of all the affected streams
           is required for the changes to take effect.
         upgradeDeadline: "2023-10-25"
+      2.0.0:
+        message:
+          Version 2.0.0 updates schemas for all hourly reports (end in report_hourly), Accounts, Campaigns, Search Query Performance Report, AppInstallAds, AppInstallAdLabels, Labels, Campaign Labels, Keyword Labels, Ad Group Labels, Keywords, and Budget Summary Report.
```

Comment:
(end in report_hourly), and the following streams: Accounts,

Reply:
updated